cmd/mdatagen/internal/samplereceiver/generated_component_telemetry_test.go

```diff
@@ -0,0 +1,72 @@
+// Code generated by mdatagen. DO NOT EDIT.
+
+package samplereceiver
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata/metricdatatest"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/receiver"
+	"go.opentelemetry.io/collector/receiver/receivertest"
+)
+
+type componentTestTelemetry struct {
```

Comment:
s/componentTestTelemetry/testTelemetry?

Reply:
i called it componentTestTelemetry to avoid conflict with places in the code where testTelemetry is already defined. I plan on renaming this to testTelemetry once the other code has been renamed. Let me open an issue about it

Reply:
https://github.com/open-telemetry/opentelemetry-collector/issues/10228